x-pack/plugins/lens/public/visualizations/partition/suggestions.ts

```diff
@@ -120,7 +120,7 @@ export function suggestions({
     const newShape = getNewShape(groups, subVisualizationId as PieVisualizationState['shape']);
     const baseSuggestion: VisualizationSuggestion<PieVisualizationState> = {
       title: i18n.translate('xpack.lens.pie.suggestionLabel', {
-        defaultMessage: 'As {chartName}',
```

Comment:
ℹ️ Some charts were using the `As ...` format and other just the visualization name. I decided to keep the later for consistency (the title is depicted on the suggestions dropdown)